wandb/sdk/launch/launch.py

```diff
@@ -99,7 +99,7 @@ def _run(
     project: Optional[str],
     entity: Optional[str],
     docker_image: Optional[str],
-    entry_point: Optional[str],
+    entry_point: Optional[List[str]],
```

Comment:
Should this also accept an `EntryPoint` type object?

Reply:
Not right now I think, at this point entry_points are just lists of strings, EntryPoint objects will be primarily used internally